Week2/6.Arrays.js

```diff
@@ -0,0 +1,23 @@
+// 6.1
+let animals = [] ;
+
+// 6.2
+console.log("the value of my array : Samoyed dog, British Shorthair, Horse") ;
+
+// 6.3
+console.log(animals) ;
+
+// 6.4
+let myFavoriteAnimals = ['Samoyed dog' , 'British Shorthair' , 'Horse'] ;
+
+// 6.5
+console.log(myFavoriteAnimals) ;
+
+// 6.6
+myFavoriteAnimals.push("Daan's favorite animal ('baby pig')") ;
+//  or we can add it as a new variable 
+// let daansFavoriteAnimal = ['baby pig'] ;
+// myFavoriteAnimals.push(daansFavoriteAnimal[0]);
```

Comment:
Nice! You can also just do `myFavoriteAnimals.push('baby pig')` that's a bit easier!